### ISSUE TITLE
Some UI elements in iOS app do not have an accessible label, have incorrect labels or missing traits 

### ISSUE THREAD

**Issue by stevefaulkner** [gitterHQ/gitter#780]
![gitterguy](https://cloud.githubusercontent.com/assets/835859/8766157/aed64434-2e24-11e5-91a3-8ce8de1d80c9.PNG)
The 'gitter guy' button in the all conversations view does not have an accessible label.
how to fix - http://www.bbc.co.uk/guidelines/futuremedia/accessibility/mobile/text-equivalents/alternatives-for-non-text-content
Also the 'gitter guy' button changes to display update notices, but this info is not exposed accessibly
![updates](https://cloud.githubusercontent.com/assets/835859/8766287/5602e74e-2e2a-11e5-974c-2841d1d0e1ca.PNG)
advice on [making dynamic content accessible](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/iPhoneAccessibility/Making_Application_Accessible/Making_Application_Accessible.html#//apple_ref/doc/uid/TP40008785-CH102-SW25) 

The home and gitter guy buttons have incorrect labels
![buttons](https://cloud.githubusercontent.com/assets/835859/8765984/8184b92a-2e1f-11e5-83a8-a19a0d5fd97c.PNG)
current 'icon home' recommended 'home'
current 'icon gitter guy' recommended *brief descriptive phrase* for function of button

The organization icons have labels such as '34ret' these icons are decorative and should have null labels. how to fix http://www.bbc.co.uk/guidelines/futuremedia/accessibility/mobile/text-equivalents/decorative-content
![organi](https://cloud.githubusercontent.com/assets/835859/8765969/c066df5c-2e1e-11e5-96e1-2580886b4bed.PNG)
The 'all conversations' list items are interactive elements, (activating a item takes you to the listed channel),but are not exposed as  such, suggest each item be given a 'button' trait. Refer to http://www.bbc.co.uk/guidelines/futuremedia/accessibility/mobile/text-equivalents/roles-and-traits 
![conversations](https://cloud.githubusercontent.com/assets/835859/8766115/385e06bc-2e23-11e5-91d3-55b67b7b1a7d.PNG)

**Comment by trevorah** [comment]
@stevefaulkner we've just release v3.0 that replaces much of the existing ui. Are there any parts that we have missed?

**Comment by stevefaulkner** [comment]
@trevorah will update app and check

**Comment by trevorah** [comment]
Thanks!

**Comment by MichielBijl** [comment]
Native all the elements, nice!

User profile image next to the text field is announced as “54567, image”. Other than that, points made by Steve seem fixed.

Another issue that is not listed here is that there is no back button in the channel list. But that is a separate issue.

**Comment by trevorah** [comment]
@MichielBijl is the confusing image announcement in the channel list? Also, the channel list is now the root view, so there is nothing to go back to.

**Comment by MichielBijl** [comment]
@trevorah no, in the chat window:

![Screenshot of bottom half of Gitter 3.0.0 iOS app with green square around user profile icon](https://cloud.githubusercontent.com/assets/5457769/12488039/03f8e318-c06a-11e5-8bd4-d6ae7d722033.png)

**Comment by trevorah** [comment]
@MichielBijl ah, thanks

**Comment by trevorah** [comment]
Ah, sounds like its reading out the filename for your avatar (https://avatars.githubusercontent.com/u/5457769?v=3). How about an `alt` with your username?

**Comment by MichielBijl** [comment]
That is what's used for the avatars in the stream, but not sure it is needed next to text area. I think @stevefaulkner would be a more appropriate person to ask.

**Comment by trevorah** [comment]
For now, i've changed it so that the avatars no longer read out anything, except for the group of avatars in the top right that dont have an accompanying username.

**Comment by trevorah** [comment]
Im closing this issue now that 3.0.2 is live with the fixes.

**Comment by MichielBijl** [comment]
Woohoo!
